py/desispec/ccdcalib.py

```diff
@@ -46,35 +48,62 @@ def compute_dark_file(rawfiles, outfile, camera, bias=None, nocosmic=False,
     Options:
         bias (str or list): bias file to use, or list of bias files
         nocosmic (bool): use medians instead of cosmic identification
-        scale (bool): apply scale correction for EM0 teststand data
         exptime (float): write EXPTIME header keyword; all inputs must match
+        min_vccdsec (float) : minimal time (in sec) after CCD bias voltage was turned on
+        max_temperature_diff (float) : maximal CCD temperature difference
+        reference_header (dict) : reference header that defines the valid hardware configuration (default is the most recent one)
+        save_preproc (bool) : save preprocessed images
+        output_specprod_dir (str) : specify output specprod directory used to save preproc images
```

Comment:
Commentary: this doesn't actually have to be a real specprod.  It just has to be a directory, under which the preproc files will be saved in dark_preproc/NIGHT/EXPID/ .  And since this directory is also used as *input* if those files exist, the "output" part of the name is a misnomer.  I don't immediately have a better name suggestion, though please clarify the help string to mention that it will read from there too if they exist.

Reply:
changed variable and script option name ouput-specprod-dir to preproc-dark-dir